indico/modules/events/persons/controllers.py

```diff
@@ -251,47 +249,70 @@ def _process(self):
                                                has_predefined_affiliations=Affiliation.query.has_rows())
 
 
-class RHEmailEventPersons(RHManageEventBase):
+class RHEmailEventPersonsPreview(RHManageEventBase):
+    """Preview an email with EventPersons associated placeholders."""
+
+    @use_kwargs({
+        'body': fields.String(required=True),
+        'subject': fields.String(load_default=None),
+    })
+    def _process(self, body, subject):
+        person = self.event.person_links[0]
```

Comment:
We were previously using a similar fashion. Even though we can get, generally, we're only emailing if the event has any participants (otherwise the button is disabled), otherwise, we should even get here. 
But revisiting now, we could harden it to check and throw.

Reply:
fine for me to leave it as is then.. if people sending crap manually get an error i don't mind as long as it doesn't happen through the UI